### ISSUE TITLE
Using ZeroTier over 4G LTE for in flight drone video (snowy picture)

### ISSUE THREAD

**Issue by mattgorski** [zerotier/ZeroTierOne#927]
First off I want to say this is the most user friendly zeroconf vpn I have ever used and I absolutely love it!

I am using ZeroTier to provide in flight video/telemetry over 4G LTE for long range flights on my drone.

I tested connection over my ISP using ethernet from my drone to work PC and everything works as expected over ethernet and I love the results, very clear picture.

When using 4G LTE the results are very different.  The video picture is snowy and I can not figure out why this is happening.  I have IPV4 in my apn settings and my speeds are just as good as my ethernet speeds, about 50mb down and 15mb up (even when connected to ZT).  My drone is only broadcasting at 750 kb/s so I know the speeds are optimal on 4G LTE.  Am I missing a crucial setting to make this work over cellular?  Do I need to route all traffic through ZT, open UDP port 9993 and also create a bridge on my linux server for better results?

All clients are using up to date zerotier v 1.2.12.

Drone is running on Ubuntu 16.04 ROS Kinetic with Sierra Wireless mini PCIe LTE onboard connected to Google Fi.

Ground Station is an Android Tablet with Google Fi LTE and full signal. Video over wifi on android with ZT works fine 4G LTE is the only issue for picture and packet loss.

Any pointers would be greatly appreciated fellow ZeroTier users :)

**Comment by laduke** [comment]
Signs point to the ground station Google LTE/Android tablet? Is there anyway to run `zerotier-cli listpeers` on the tablet? listpeers can tell you if your connection to another node is direct, or relaying through a zerotier inc server. See [router config tips](https://zerotier.atlassian.net/wiki/spaces/SD/pages/6815768/Router+Configuration+Tips) for more info

> Do I need to route all traffic through ZT, open UDP port 9993 and also create a bridge on my linux server for better results?

Open port 9993, yet. The others aren't necessary.

**Comment by laduke** [comment]
There's no zerotier-cli on android. Oops. 
I hear you can install zerotier in termux though, for debugging purposes.

**Comment by mattgorski** [comment]
Okay sorry for the late reply on this but I have solved (I believe) my issue with a couple different steps:

1:  Added CONFIG_IP_MULTICAST=y to my Android kernel.
2:  Installed Zero Tier on my Linux server and I used Wowza Streaming Engine/Server (Developer Licence for one stream) and fed my drone RSTP gst pipline to the wowza server with the Zero Tier IP.  This helps as the uplink can feed to multiple targets and is only streamed from the drone with minimal upload and latency.  I will report back more results soon....

**Comment by bobby8133** [comment]
@mattgorski & @laduke - Maybe you guys can help me?!? I have a raspberry pi with Navio2, I have installed Zerotier and was able to connect to my to zerotier network with a 4g usb stick.  I'm using a laptop with Mission planner  connected to my cell phone hotspot. I installed zerotier on my laptop and was able to connect to the network I created in zerotier.  In zerotier I checked the box to allow bridging. Do I have to bridge the networks on my laptop and raspberry pi?  I can see both devices online in my zerotier account.
